analytics.js

```diff
@@ -955,6 +955,10 @@ class Analytics {
     return this.anonymousId;
   }
 
+  getUserId() {
+    return this.userId;
```

Comment:
It should have been `this.storage.getUserId()` as this method can be called even prior to the `load` API.

Reply:
Would it work if i change the following? (Also figured this would align with getAnonymousId)
```
getUserId() {
    this.userId = this.storage.getUserId();
    return this.userId;}
```

Reply:
Yes, it will.

Reply:
Done